examples/samples/README.md

```diff
@@ -0,0 +1,60 @@
+## Getting started with samples ##
+
+## Building the library
+- Clone the repo and cd to the repo directory
+```bash
+git clone https://github.com/azure/azure-service-bus-node.git
+cd azure-service-bus-node
+```
+- Install typescript, ts-node globally (optional, but very useful)
+```bash
+npm i -g typescript
+npm i -g ts-node
+```
+- NPM install from the root of the package
+```bash
+npm i
+```
+- Build the project
+```bash
+npm run build
+```
+
+## Before executing a sample
+- Go to the [Azure Portal](https://portal.azure.com).
+- Here are the docs which would help you create a service bus resource in the portal : [ServiceBus - NodeJS DOCS](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-nodejs-how-to-use-queues).
+- In the portal, go to **Dashboard > Service Bus > _your-servicebus-namespace_**.
+- Copy the "Primary Connection String" of **RootManageSharedAccessKey** at **Shared access policies** under **Settings** tab.
+- To work with Queues, find the "Queues" tab right under "Entities" at **_your-servicebus-namespace_** and create a Queue and copy its name for next steps.
```

Comment:
Break this sentence up for ease of reading.

`... and create a Queue. Copy its name for next steps.` instead of `... and create a Queue and copy its name for next steps`